features/support/env.rb

```diff
@@ -7,6 +7,10 @@
 require 'cucumber/rails'
 require 'database_cleaner'
 
+require 'capybara/poltergeist'
+Capybara.javascript_driver = :poltergeist
+WebMock.allow_net_connect!
```

Comment:
Capybara with a js driver really is a separate server that is being connected to. With unit tests WebMock can make sure you are not accidentally hitting a server, but with acceptance tests, it feels disingenuous.